jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-jetty-client/pom.xml

```diff
@@ -32,6 +32,16 @@
           </instructions>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>
+            @{argLine} ${jetty.surefire.argLine}
+            --add-modules org.eclipse.jetty.ee8.webapp
+            --add-reads org.eclipse.jetty.ee8.websocket.jetty.client=org.eclipse.jetty.ee8.webapp
```

Comment:
You do not need the `read` line. That is only used where we cannot declare an outright `requires` in the `module-info.java`, eg for test-only dependencies. Your `module-info.java` has declared a `requires static` and that _must_ be sufficient and should _not_ need to be forcibly overridden in this way.